pkg/block/block_test.go

```diff
@@ -53,9 +53,6 @@ func TestBlock(t *testing.T) {
 	if d0.Name == "" {
 		t.Fatalf("Expected disk name, but got \"\"")
 	}
-	if d0.SerialNumber == "unknown" {
-		t.Fatalf("Got unknown serial number.")
-	}
```

Comment:
related?

Reply:
This was just a bad unit test that I removed. Sorry, unrelated to this PR.